### PR TITLE
Add CoordinateEditHelper Unit Tests

### PR DESCRIPTION
**Pull request by madhurgupta10**
**Description (required)**
Add CoordinateEditHelper Unit Tests

@neslihanturan @misaochan I am not super familiar with Wikidata, so please do check the passed in Wikidata and expected results in the tests.

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/coordinates/CoordinateEditHelper.java [hunk 1]
```diff
@@ -142,17 +142,7 @@ private String getFormattedWikiText(final String wikiText, final String editedLo
 
             return wikiText + editedLocation;
         }
-        if (!wikiText.contains("filedesc") && !wikiText.contains("Location")) {
```
Comment (madhurgupta10) [inline app/src/main/java/fr/free/nrw/commons/coordinates/CoordinateEditHelper.java:145]: Not sure if this was intended or a mistake, let me know 😅

Comment (misaochan) [reply app/src/main/java/fr/free/nrw/commons/coordinates/CoordinateEditHelper.java:145]: @Ayan-10 I believe you wrote this class, could you comment on this please? :)

**Review by misaochan — APPROVED** [review]
Looks good to me, happy to merge pending clarification by class author

File: app/src/main/java/fr/free/nrw/commons/coordinates/CoordinateEditHelper.java (hunk 1 above)
Comment (Ayan-10) [reply app/src/main/java/fr/free/nrw/commons/coordinates/CoordinateEditHelper.java:145]: I was doing case by case comparison at that time that's why I wrote like that. But now it seems redundant. Sorry for not recognizing it earlier.

Thanks

**Pull request merged**